ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/about-you/about-you.component.html

```diff
@@ -3,10 +3,75 @@
     <h1 class="govuk-heading-xl" i18n="@@aboutYou_h1">
       About you (the legal representative)
     </h1>
-    <app-hearing-details-header></app-hearing-details-header>
-    <div class="mt-20">
-      <button class="govuk-button" (click)="continue()" i18n="@@aboutYou_btn_continue" id="continue"
-        type="button">Continue</button>
-    </div>
+
+    <form [formGroup]="form" (ngSubmit)="continue()">
+
+      <div id="form-container" [ngClass]="isFormInvalid && !isTextInputInvalid ? 'govuk-form-group--error' : 'govuk-form-group'">
+        <div class="govuk-fieldset">
+          <div *ngIf="isFormInvalid && !isTextInputInvalid">
+            <span class="govuk-error-message" i18n="@@aboutYou_span_1">
+              Please answer this question
+            </span>
+          </div>
+
+          <p class="govuk-body govuk-!-margin-bottom-6" i18n="@@aboutYou_p_1">
+            Is there anything that could affect your ability to take part in a video hearing?
+          </p>
+
+          <p class="govuk-body govuk-!-margin-bottom-1" i18n="@@aboutYou_p_2">
+            For example:
+          </p>
+
+          <ul class="govuk-list govuk-list--bullet">
+              <li i18n="@@aboutYou_listItem_1">are you hard of hearing or deaf?</li>
+              <li i18n="@@aboutYou_listItem_2">are you blind or partially sighted?</li>
+              <li i18n="@@aboutYou_listItem_3">are there any other factors that might make communicating by video difficult?</li>
+          </ul>
+
+          <div class="govuk-form-group">
+            <div class="govuk-fieldset fieldset-margin">
+              <div class="govuk-radios">
+
+                <div class="govuk-radios__item">
+                  <input formControlName="choice" class="govuk-radios__input" id="choice-yes" type="radio"
+                         [value]="true">
+                  <label class="govuk-label govuk-radios__label" for="choice-yes" i18n="@@aboutYou_label_1">
+                    Yes
+                  </label>
+                </div>
+                <div [ngClass]="isTextInputInvalid ? 'vh-radios__conditional--error' : 'govuk-radios__conditional'">
+                  <div class="form-group textarea-top-margin" *ngIf="choice.value">
+                    <div>
+                      <label class="govuk-label" i18n="@@aboutYou_label_1">
+                        Please tell us more about that
+                      </label>
+                      <span *ngIf="isTextInputInvalid" class="govuk-error-message" i18n="@@aboutYou_span_2">
+                        Please type in the box
+                      </span>
+                    </div>
+                    <textarea formControlName="textInput" aria-label="option-yes" class="txtMore vh-text-width" id="details" rows="5" cols="80"
```

Comment:
I think the `aria-label` here need to be something else, it could also have a `aria-describedby=` and link to the label above, if you give that an id

Reply:
https://www.w3.org/TR/wai-aria-1.1/#aria-describedby

Reply:
Sorted now